site/pages/docs/ref/exitscript/exitscript-en.hbs

```diff
@@ -54,6 +54,12 @@
 				<td>Add <code>"i18n":{ "exitMsg": "This is a custom message."}</code> attribute.</td>
 				<td>Your custom message</td>
 			</tr>
+			<tr>
+				<td><code>targetWarning</code></td>
+				<td>Show custom message</td>
+				<td>Add <code>"i18n":{ "targetWarning": "This is a custom message."}</code> attribute.</td>
+				<td>Your custom message</td>
+			</tr>
```

Comment:
@ipaksc I am about to merge changes from @duboisp that were made before your PR and covers a little bit more (it has French changes for instance). https://github.com/wet-boew/wet-boew/pull/8978

As soon as I merge it, can you please rebase your work to the upstream master, fix conflicts and force push; else it would be even easier to just reset hard and do a PR only for those lines that you added there? In English and in French.

`<tr>
				<td><code>targetWarning</code></td>
				<td>Show custom message</td>
				<td>Add <code>"i18n":{ "targetWarning": "This is a custom message."}</code> attribute.</td>
				<td>Your custom message</td>
			</tr>`

Thanks!